zadani/soft/style.css

```diff
@@ -11,33 +13,166 @@
 body {
     margin: 0;
     padding: 0;
-    background-color: #f5f5f5;
     font-family: 'Merriweather', serif;
     font-size: 16px;
     line-height: 1.6;
+    background-color: #f5f5f5;
 }
 
-
 /* HLAVICKA */
 
+.hlavicka {
+    display: flex;
+    flex-direction: column;
+    text-align: center;
+    background-image: url('images/hlavicka.jpg');
+    /* proc se nezobrazuje???*/
+    min-width: 500px;
+}
+
 .titulek {
     font-family: 'Raleway', serif;
+    font-size: 3rem;
+}
+
+.autor {
+    font-style: italic;
+    border-top: 5px solid white;
+    padding: 1rem;
 }
 
 
 /* MENU */
 
+.menu-obsah {
+    display: flex;
+    flex-direction: column;
+    min-width: 500px;
```

Comment:
Minimální šířku určitě definovat nechceme, budeme chtít jen u PC určit naopak maximální šířku